### ISSUE TITLE
Motor assistance does not resume after standstill when lights are on

### ISSUE THREAD

**Issue by j0bro** [OpenSource-EBike-firmware/TSDZ2-Smart-EBike#94]
I installed lights and now when I have them on, the motor does not resume assistance after you had a standstill. It looks like you need to keep pedal cadence going, or some speed or power output on the motor, I don't know what makes it stop exactly.

After I switch off the lights, motor assistance kicks in again when pedaling.

Firmware 0.18.2

**Comment by j0bro** [comment]
Is there any workaround possible for this? Thanks in advance @casainho and @leon927

**Comment by leon927** [comment]
The best thing to do is to just wait for the stable 0.19.0 that is soon to be released. And if you still experience problems after updating it will be sorted out. 

If you can not wait for the official release I can recommend the 0.19.0 beta 8. No known bugs and has a lot of great changes.

**Comment by j0bro** [comment]
I found this issue has been (re)visited already a few times in, but for me, the fix as proposed in closed issue #71 does not work. Should we maybe have a configurable lights current to allow users to tweak this behaviour?

Update: OK will wait for the 0.19 stable release. Any idea when it will be put out? I saw a warning on the previous betas to not use it, only for testing so I was not sure if it is actually usable for day to day operation.
